src/resource_clients/actor.ts

```diff
@@ -330,25 +330,46 @@ export interface ActorCallOptions extends Omit<ActorStartOptions, 'waitForFinish
     waitSecs?: number;
 }
 
-export interface ActorRun {
+export interface ActorRunList {
```

Comment:
This name suggests that this interface describes the list of actor runs, but this is still just a single run.

How about calling this `ActorRunListItem`?